benchmarks/hyperspecialization/list-generate/generate_data.py

```diff
@@ -0,0 +1,52 @@
+# condition on a) type, b) length, c) uniqueness of elements
+import argparse
+import pickle
+import random
+import pdb
+import string
+
+def randint():
```

Comment:
how do we influence the distribution? I think just doing complete randomness will lead to non many duplicate values. Yet, in real datasets often we have low cardinalities. Thus, I'd recommend to have some option to generate such things. => some distribution parameter maybe?